### PR TITLE
Introduced setting log level during Ankaios installation

### PR DESCRIPTION
**Pull request by lingnoi**
Issues: #163

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [ ] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

File: tools/install.sh [hunk 1]
```diff
@@ -122,6 +123,26 @@ else
     ANKAIOS_RELEASE_URL_SHA="${RELEASE_URL_BASE}/download/${ANKAIOS_VERSION}/${RELEASE_FILE_NAME_WITH_SHA}"
 fi
 
+if [ -z "$INSTALL_ANK_SERVER_RUST_LOG" ] ; then
+    echo "No log level for ank-server provided, use default: 'info'"
+    INSTALL_ANK_SERVER_RUST_LOG=${DEFAULT_LOG_LEVEL}
+else
+    if [[ ! "${INSTALL_ANK_SERVER_RUST_LOG}" =~ ^(info|debug|warn|error|trace)$ ]]; then
```
Comment (windsource) [inline tools/install.sh:130]: Why should we restrict the content of the log level? I think the user should have all freedom including settings different log levels per component (see [documentation](https://docs.rs/env_logger/latest/env_logger/#enabling-logging)). So we should accept all values and just pass them to `RUST_LOG` with quotes.

File: doc/docs/usage/installation.md [hunk 2]
```diff
@@ -69,6 +69,30 @@ curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>
 
 For available versions see the [list of releases](https://github.com/eclipse-ankaios/ankaios/tags).
 
+### Set the log level for `ank-server` and `ank-agent` services
+
+To configure the log levels for `ank-server` and `ank-agent` during the installation process using the provided environment variables, follow these steps:
+
+1. Set the desired log levels for each service by assigning valid values to the environment variables `INSTALL_ANK_SERVER_RUST_LOG` and `INSTALL_ANK_AGENT_RUST_LOG`. Valid values include `info`, `debug`, `warn`, `error`, or `trace`. For example:
+   * To set the log level for `ank-server` to "debug": `INSTALL_ANK_SERVER_RUST_LOG=debug`
+   * To set the log level for `ank-agent` to "info": `INSTALL_ANK_AGENT_RUST_LOG=info`
```
Comment (windsource) [inline doc/docs/usage/installation.md:78]:
```suggestion
1. Set the desired log levels for each service by assigning valid values to the environment variables `INSTALL_ANK_SERVER_RUST_LOG` and `INSTALL_ANK_AGENT_RUST_LOG`. For the syntax see the [documentation for `RUST_LOG`](https://docs.rs/env_logger/latest/env_logger/#enabling-logging).
```

File: doc/docs/usage/installation.md [hunk 3]
```diff
@@ -69,6 +69,28 @@ curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>
...
+For a specific version:
+
+```shell
+      curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>/install.sh | INSTALL_ANK_SERVER_RUST_LOG=debug INSTALL_ANK_AGENT_RUST_LOG=info bash -s -- -i /usr/local/bin -t both -v <version>
+```
+
+For the latest version:
+
+```shell
+    curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/latest/install.sh | INSTALL_ANK_SERVER_RUST_LOG=debug INSTALL_ANK_AGENT_RUST_LOG=info bash -s -- -i /usr/local/bin -t both
```
Comment (windsource) [inline doc/docs/usage/installation.md:89]:
```suggestion
curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/latest/install.sh | INSTALL_ANK_SERVER_RUST_LOG=debug INSTALL_ANK_AGENT_RUST_LOG=info bash -s -- -i /usr/local/bin -t both
```

File: doc/docs/usage/installation.md [hunk 4]
```diff
@@ -69,6 +69,28 @@ curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>
...
+To configure the log levels for `ank-server` and `ank-agent` during the installation process using the provided environment variables, follow these steps:
+
+1. Set the desired log levels for each service by assigning valid values to the environment variables `INSTALL_ANK_SERVER_RUST_LOG` and `INSTALL_ANK_AGENT_RUST_LOG`. For the syntax see the [documentation for `RUST_LOG`](https://docs.rs/env_logger/latest/env_logger/#enabling-logging).
+
+2. Run the installation script, making sure to pass these environment variables as arguments if needed:
+
+For a specific version:
+
+```shell
+      curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>/install.sh | INSTALL_ANK_SERVER_RUST_LOG=debug INSTALL_ANK_AGENT_RUST_LOG=info bash -s -- -i /usr/local/bin -t both -v <version>
```
Comment (windsource) [inline doc/docs/usage/installation.md:83]:
```suggestion
curl -sfL https://github.com/eclipse-ankaios/ankaios/releases/download/<version>/install.sh | INSTALL_ANK_SERVER_RUST_LOG=debug INSTALL_ANK_AGENT_RUST_LOG=info bash -s -- -i /usr/local/bin -t both -v <version>
```

**Review by windsource — CHANGES_REQUESTED** [review]
See comments.

File: tools/install.sh (hunk 1 above)
Comment (lingnoi) [reply tools/install.sh]: Done.

**Review by windsource — APPROVED** [review]
OK

**Pull request merged**